pkg/printers/internalversion/printers.go

```diff
@@ -741,6 +742,7 @@ func printPod(pod *api.Pod, options printers.GenerateOptions) ([]metav1.TableRow
 	restarts := 0
 	totalContainers := len(pod.Spec.Containers)
 	readyContainers := 0
+	lastRestartDate := metav1.Date(1, 1, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
`metav1.NewTime(time.Time{})` is simpler

Reply:
I was looking for something like this, thank you!